modules/apps/account/account-test/src/testFunctional/tests/2BAccount.testcase

```diff
@@ -28,6 +28,12 @@ definition {
 			accountEntryName = "Account Name",
 			accountEntryType = "Business");
 
+		Accounts.openAccountsAdmin();
+
+		LexiconEntry.gotoEntry(rowEntry = "Account Name");
+
+		Accounts.addOwnedAccountRole(accountRoleTitle = "Account Specific Role");
```

Comment:
Not adding the role in set-up since it's not a common step for all tests in the test file.